srml/contract/src/lib.rs

```diff
@@ -122,38 +122,38 @@ pub trait ComputeDispatchFee<Call, Balance> {
 }
 
 #[derive(Encode,Decode,Clone,Debug)]
-/// Information for managing an acocunt and its sub trie abstraction.
-/// This is the required info to cache for an account
+/// Information for managing an account and its sub trie abstraction.
+/// This is the required info to cache for an account.
 pub struct AccountInfo {
-	/// unique ID for the subtree encoded as a byte
+	/// Unique ID for the subtree encoded as a byte.
 	pub trie_id: TrieId,
-	/// the size of stored value in octet
+	/// The size of stored value in octet.
 	pub storage_size: u64,
 }
 
-/// Get a trie id (trie id must be unique and collision resistant depending upon its context)
-/// Note that it is different than encode because trie id should have collision resistance
-/// property (being a proper uniqueid).
+/// Get a trie id (trie id must be unique and collision resistant depending upon its context).
+/// Note that it is different than encode because trie id should be collision resistant
+/// (being a proper uniqueid).
 pub trait TrieIdGenerator<AccountId> {
```

Comment:
A bit out of scope, but I'm not sure this should be named "Trie". The trie is an abstraction level beyond storage and is an implementation detail as far as the runtime module is concerned. In this case, it's specifically the `AccountId` that must be a unique identifier, so it should probably be called `AccountIdGenerator`

cc @pepyakin